misc/pytorch_toolkit/breast_ultrasound_simulation/src/exporter.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from .model import GeneratorModel
+import argparse
+import os
+import subprocess
+
+def export_model_onnx(config_args):
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+    print(torch.cuda.is_available())
```

Comment:
Either delete this or make it more informative (like `print(f'torch CUDA is available: {torch.cuda.is_available()}')`)